lib/FlashMem/FlashMem.cpp

```diff
@@ -151,23 +173,39 @@ bool Flash::saveString(const uint8_t &address, const uint8_t &maxLength, const S
     return isSuccess;
 }
 
-bool Flash::setHeader(bool areCredentialsStored)
+bool Flash::fetchInt(const uint8_t &address, uint8_t &value)
```

Comment:
Naming inconsistent, use read/write or get/set, but not fetch and save.
And you don't read a int here, as the name says ... you read a uint8_t.